requirements_devel.txt

```diff
@@ -7,5 +7,72 @@
 
 # Development packages
 
-django-debug-toolbar
+
 coverage
+alabaster==0.7.10
```

Comment:
There seems to be quite a number of packages not related to this task. please clean this up